conda/recipes/libkvikio/recipe.yaml

```diff
@@ -82,10 +83,12 @@ outputs:
       host:
         - cuda-version =${{ cuda_version }}
         - libcurl ${{ libcurl_version }}
+        - libnuma
       run:
         - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
```

Comment:
I don't know why the `libnuma` run export isn't showing up -- I'll try to come up with a smaller reproducer.  Could be something with outputs that pin themselves as their own run exports?

In any case, since it isn't showing up in the build logs, we could explicitly add this under `run` here to make sure we get the dependencies needed at runtime:

```yaml
${{ pin_compatible("libnuma", upper_bound="x") }}
```

Reply:
Ok, yeah, the `libnuma` feedstock recipe is just wrong.  I'll try to upstream the fixes, but for now, I would add an explicit runtime dep

Reply:
Good call. Done in b239a211f53bdf45d008e4125ef7bb2480b18ea6. I'll leave this open for now. Here's the upstream PR: https://github.com/conda-forge/numactl-feedstock/pull/18